packages/core/src/components/Tooltip.svelte

```diff
@@ -98,21 +159,94 @@
     .backgroundColor}"
 >
   <div class="visahoi-tooltip-title">
-    <b>{$activeMarker?.tooltip.title}</b>
-    <div class="visahoi-close-tooltip" on:click={closeTooltip}>
+    {#if editTooltip}
+      <input class="visahoi-edit-title" type="text" bind:value={tempTitle} />
+    {:else}
+      <b>{$activeMarker?.tooltip.title}</b>
+    {/if}
+
+    <!--The trash icon is shown in the tooltip only isEditModeActive is set to true-->
+    {#if $isEditModeActive && !editTooltip}
```

Comment:
The buttons in edit mode should always be on the right side (always in the same place, no matter how long the title is). 

Therefore I'd suggest to wrap the icons into another div element, so that the icons are still on the right side of the tooltip header, and the 'justify-content: space-between' on the parent element just adds some space between the title text and the icons. Then, add some space between the icons (i'd say around 2px).